### DISCUSSION TITLE
Issues with Conda environment for MOOSE

### DISCUSSION THREAD

**Discussion by hityyds** [idaholab/moose#26433] [Q&A General]
Recently, I encountered some problems while installing gperftools for MOOSE code. This not only prevented me from using gperftools but also disrupted my Conda MOOSE environment. Now, when compiling my program, I am encountering the following error:
```
moose/framework/contrib/hit/parse.cc:687:34: error: 'class wasp::HITNodeView' has no member named 'token_type'; did you mean 'set_type'?
  687 |     auto token_type = value_node.token_type();
      |                                  ^~~~~~~~~~
      |                                  set_type
```
As a result, I plan to reinstall MOOSE, but Conda is having difficulties installing the MOOSE environment. Initially, during the installation of moose-tools, Conda would consume over 30GB of memory, leading to the forced termination of Conda by the system. After trying some suggestions found online, Conda is now throwing the following error during the installation of moose-dev and moose-tools:
```
Collecting package metadata (current_repodata.json): done
Solving environment: unsuccessful initial attempt using frozen solve. Retrying with flexible solve.
Solving environment: unsuccessful attempt using repodata from current_repodata.json, retrying with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: unsuccessful initial attempt using frozen solve. Retrying with flexible solve.
Solving environment: | 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                    

UnsatisfiableError: 
```
My Python version is 3.10.12.

**Comment by GiudGiud** [comment]
Hello

I would delete conda and try again with our new miniforge install route
https://mooseframework.inl.gov/getting_started/installation/conda.html

The first error was WASP out of sync. The last one is a conda problem and there's no other way around it that I know.

Guillaume

**Reply by hityyds** [reply]
I successfully resolved the issue. Previously, in order to address some problems related to conda, I followed online advice and set the conda solver to "classic." After removing this command from the .condarc file, conda was able to install moose-dev successfully.

**Reply by hityyds** [reply]
Thank you very much for your attention and advice.
